sklearn/tree/export.py

```diff
@@ -142,6 +144,10 @@ def export_graphviz(decision_tree, out_file=SENTINEL, max_depth=None,
         When set to ``False``, ignore special characters for PostScript
         compatibility.
 
+    decimals : int, optional (default=3)
+        The number of decimal reported of the ``impurity``,
```

Comment:
Desired precision in the values of ``impurity``, ``threshold`` and ``value`` attributes of each node.